README.md

```diff
@@ -1,49 +1,92 @@
 # pay-selfservice
 GOV.UK Pay Self Service admin tool (Node.js)
 
-## Running locally with `pay local`
+## Prerequisites
+* This requires the [Pay CLI](https://github.com/alphagov/pay-infra/tree/master/cli), which is not publicly available at present.
+* You have [set up your local development environment](https://pay-team-manual.cloudapps.digital/manual/setup-local-dev-environment.html)
+* Clone this repo locally.
 
-*This requires the [Pay CLI](https://github.com/alphagov/pay-infra/tree/master/cli), which is not publicly available at present*
+### Running
 
-### Prerequisites
-You have [set up your local development environment](https://pay-team-manual.cloudapps.digital/development-processes/setup-local-dev-environment/)
+Self service reads from environment variables that allow it to connect to backend services from any environment. There are scripts to create an environment file to point to different backends.
```

Comment:
Can we remove `There are scripts to create an environment file to point to different backends.` or elaborate where these scripts are ?

Reply:
I removed the line - I think it reads better with less text.